Example/SBTUITestTunnel/SBTAppDelegate.m

```diff
@@ -25,16 +25,20 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     [SBTUITestTunnelServer registerCustomCommandNamed:@"myCustomCommandReturnNil" block:^NSObject *(NSObject *object) {
         [[NSUserDefaults standardUserDefaults] setObject:object forKey:@"custom_command_test"];
-        [[NSUserDefaults standardUserDefaults] synchronize];
```

Comment:
not needed according to [this](https://developer.apple.com/documentation/foundation/nsuserdefaults/1414005-synchronize?language=objc):

```
Waits for any pending asynchronous updates to the defaults database and returns; this method is unnecessary and shouldn't be used.
```

I also confirmed tests are passing locally